engine/src/main/scala/cromwell/services/EngineMetadataServiceActor.scala

```diff
@@ -23,6 +45,34 @@ case class EngineMetadataServiceActor(serviceConfig: Config, globalConfig: Confi
     }
   }
 
+  private def queryStatusAndRespond(id: WorkflowId): Unit = {
+    val sndr = sender()
+    dataAccess.getWorkflowStatus(id) onComplete {
```

Comment:
We only have one DB/schema. #863 is the ticket to change that.  This is hitting the summary table.